source/aggregation-pipeline-builder.txt

```diff
@@ -370,6 +374,40 @@ To create a view from your pipeline results:
 
 .. include:: /includes/fact-create-view-from-pipeline.rst
 
+.. _refresh-document-counts-after-deletes:
+
+Refreshing Document Counts after Delete Operations
+--------------------------------------------------
+
+When running an aggregation pipeline, the :guilabel:`count results` 
+number may not be accurate after a deletion operation. To see an 
+accurate count of a collections documents after a delete has processed:
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click the :guilabel:`Edit` button.
+
+      .. figure:: /images/compass/agg-builder-click-edit-button.png
+         :alt: Click the edit button
+
+   .. step:: Click :guilabel:`Run` button.
+
+      .. figure:: /images/compass/agg-builder-click-run-button.png
+         :alt: Click the run button
+
+   .. step:: Click the :guilabel:`count results` link. 
+
+      .. figure:: /images/compass/agg-builder-click-count-results.png
+         :alt: Click the count results link
+
+      |compass-short| updates the :guilabel:`count results` number in the 
```

Comment:
Consider shortening:
 :guilabel:`count results` displays the number of documents in the collection. 


Alternatively, consider replacing "correct number" with "updated number" to avoid 
the suggestion of bad data.